service/src/main/java/uk/gov/mca/beacons/api/export/xlsx/backup/BackupSpreadsheetRow.java

```diff
@@ -149,7 +149,7 @@ public BackupSpreadsheetRow(
     ExportMapper exportMapper,
     DateTimeFormatter dateFormatter
   ) {
-    BeaconExportDTO mappedBeacon = exportMapper.toBeaconBackupExportDTO(
+    BeaconExportDTO mappedBeacon = exportMapper.toBeaconExportDTO(
```

Comment:
We do still need a separate exportMapper.toBeaconBackupExportDTO() method. This is because the uses are mapped in an entirely different way to how they're being mapped in exportMapper.toBeaconExportDTO(). Please can you reinstate toBeaconBackupExportDTO()? 🙏